modules/tiles/src/tileset/tile-3d.ts

```diff
@@ -296,15 +296,19 @@ export default class TileHeader {
     /*
      * Tiles that are outside of the camera's frustum could be skipped if we are in 'ADD' mode
      * or if we are using 'Skip Traversal' in 'REPLACE' mode.
-     * In 'REPLACE' and 'Base Traversal' mode, all child tiles have to be loaded and displayed,
-     * including ones outide of the camera frustum, so that we can hide the parent tile.
+     * Otherewise, all 'touched' child tiles have to be loaded and displayed,
+     * this may include tiles that are outide of the camera frustum (so that we can hide the parent tile).
      */
     const maySkipTile = this.refine === TILE_REFINEMENT.ADD || skipLevelOfDetail;
 
     // Check if any reason to abort
     if (maySkipTile && !this.isVisible && this._visible !== undefined) {
       return -1;
     }
+    // Condition used in `cancelOutOfViewRequests` function in CesiumJS/Cesium3DTileset.js
+    if (this.tileset._frameNumber - this._touchedFrame >= 1) {
```

Comment:
If we are uncertain we could make this controllable with a `Tileset3D` option.

Reply:
Well, I haven't found a case yet where it doesn't help so would be good if we can have it as a default. Should we maybe have a list of typical i3s/3d-tiles use-cases that we can at least test manually after such traversal changes?